src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -593,6 +597,17 @@ zocl_xclbin_read_axlf(struct drm_zocl_dev *zdev, struct drm_zocl_axlf *axlf_obj)
 		ret = zocl_load_aie_only_pdi(zdev, axlf, xclbin);
 		if (ret)
 			goto out0;
+	} else if ( axlf_obj->za_flags == DRM_ZOCL_PLATFORM_FLAT && 
```

Comment:
you don't need space between "if (" and "axlf_obj."

Reply:
fixed